docs/sources/auth/azuread.md

```diff
@@ -110,23 +115,28 @@ allowed_groups =
 
 > Note: Ensure that the [root_url]({{< relref "../administration/configuration/#root-url" >}}) in Grafana is set in your Azure Application Reply URLs (App -> Settings -> Reply URLs)
 
-### Configure allowed groups and domains
+### Configure allowed groups
 
-To limit access to authenticated users that are members of one or more groups, set `allowed_groups`
-to a comma- or space-separated list of group Object Ids. Object Id for a specific group can be found on the Azure portal: go to Azure Active Directory -> Groups. For instance, if you want to
-only give access to members of the group `example` which has Id `8bab1c86-8fba-33e5-2089-1d1c80ec267d`, set
+To limit access to authenticated users who are members of one or more groups, set `allowed_groups`
+to a comma- or space-separated list of group Object Ids. Object Ids for a specific group appear on the Azure portal:
```

Comment:
The capitalization of "Object Ids" is confusing. Is this in an Azure UI somewhere? If not, then "object IDs".